control_plane/src/endpoint.rs

```diff
@@ -957,6 +1006,10 @@ impl Endpoint {
                 self.external_http_address.port()
             ))
             .header(CONTENT_TYPE.as_str(), "application/json")
+            .header(
```

Comment:
Maybe we should use a .bearer()? (I've used that in compute_ctl changes)

Reply:
Let see about this. Haven't heard of it.